src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilarIssues/similarTraceID/body.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {Location} from 'history';
+import moment from 'moment-timezone';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {getTraceDateTimeRange} from 'app/components/events/interfaces/spans/utils';
+import EventView from 'app/utils/discover/eventView';
+import {t} from 'app/locale';
+import {ALL_ACCESS_PROJECTS} from 'app/constants/globalSelectionHeader';
+import {Organization, Event} from 'app/types';
+import DiscoverQuery from 'app/utils/discover/discoverQuery';
+import {Panel, PanelBody} from 'app/components/panels';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+
+import List from './list';
+
+type Props = {
+  event: Event;
+  organization: Organization;
+  location: Location;
+  traceID?: string;
+};
+
+const Body = ({traceID, organization, event, location}: Props) => {
+  if (!traceID) {
+    return (
+      <Panel>
+        <PanelBody>
+          <EmptyStateWarning small withIcon={false}>
+            {t(
+              'This event has no trace context, therefore it was not possible to fetch similar issues by trace ID.'
+            )}
+          </EmptyStateWarning>
+        </PanelBody>
+      </Panel>
+    );
+  }
+
+  const orgSlug = organization.slug;
+  const orgFeatures = organization.features;
+  const dateCreated = moment(event.dateCreated).valueOf() / 1000;
+  const {start, end} = getTraceDateTimeRange({start: dateCreated, end: dateCreated});
+
+  const eventView = EventView.fromSavedQuery({
+    id: undefined,
+    name: `Issues with Trace ID ${traceID}`,
+    fields: ['issue.id'],
+    orderby: '-timestamp',
+    query: `trace:${traceID} !event.type:transaction !id:${event.id} `,
```

Comment:
Instead of filtering by event.id, should it be by issue id so that the current issue is not included? Otherwise you could have other events in the current issue.

Reply:
The idea is to display all events of an issue that have the same trace ID. So I believe the logic is correct